cmd/containerd-nydus-grpc/app/snapshotter/main.go

```diff
@@ -28,5 +29,6 @@ func Start(ctx context.Context, cfg config.Config) error {
 	opt := ServeOptions{
 		ListeningSocketPath: cfg.Address,
 	}
+	auth.InitKubeSecretListener(ctx, cfg.KubeconfigPath)
```

Comment:
`InitKubeSecretListener` does not tell its failure. Can tolerate its failure even the option `--kubeconfig-path` is given?